source/js/components/join/join.jsx

```diff
@@ -21,6 +21,31 @@ class JoinUs extends Component {
       privacyCheckbox: utility.generateUniqueId(`join-privacy-checkbox`),
     };
     this.state = this.getInitialState(props);
+
+    // Used for toggle visibility for buyersguide newsletter container
+    if (document.querySelector("#buyersguide_newsletter_box")) {
+      document
+        .querySelector("#buyersguide_newsletter_box")
```

Comment:
Have you considered storing the found element in a variable? I think this could aid readability.